.github/workflows/ci.yml

```diff
@@ -19,5 +19,12 @@ jobs:
       - name: Make ZIP files
         run: ./make_zip_files.sh
 
-      - name: Run addons-linter
-        run: find dist -type f -name '*.zip' -print0 | xargs -0 -n1 sh -c 'echo "$0" ; npx addons-linter "$0"'
+      - name: Run addons-linter on regular extensions
+        run: find dist/regular -type f -name '*.zip' -print0 | xargs -0 -n1 sh -c 'echo "$0" ; npx addons-linter --self-hosted "$0"'
+
+      - name: Run addons-linter on privileged extensions
+
+        run: |
+          # dist/privileged/privileged.zip is allowed to fail because it has a `http` update URL
```

Comment:
As we were discussing over chat it is annoying that we may not need the update_url in the end, but at the same time this is a small detail and not sure it is worth spending too much time on it, we can decide to followup on it eventually later (e.g. by giving a try to remove the update_url all together or setting it to an https url and give it a try push).

Reply:
I removed `update_url` (and all the hacks)